.changeset/soft-badgers-obey.md

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
issue (blocking) we don't version the showcase or the website, so this file can be deleted